l2tdevtools/review_helpers/github.py

```diff
@@ -10,7 +10,8 @@
 from l2tdevtools.lib import errors
 
 
-class GitHubHelper(object):
+# Python3 linter complains, need to keep for python2.
```

Comment:
Let's move this to pylint config file, rather than doing in item for item - we'll need it in the pylint templates too

Reply:
ack